src/rinterface_extra.c

```diff
@@ -620,7 +650,7 @@ igraph_error_t R_igraph_attribute_add_vertices(igraph_t *graph, igraph_integer_t
     val=VECTOR_ELT(attr, 2);
     UNPROTECT(9);
   }
-  igraph_vector_int_destroy(&news);
+  igraph_vector_int_destroy_pv(&news);
```

Comment:
I don't think this is necessary, the `igraph_vector_int_destroy()` function is best here. I'll revert that part.